signac_dashboard/dashboard.py

```diff
@@ -375,19 +385,32 @@ def _setup_pagination(self, jobs):
 
     def _render_job_view(self, *args, **kwargs):
         g.active_page = "jobs"
+        session["context"] = "JobContext"
         view_mode = request.args.get("view", kwargs.get("default_view", "list"))
         if view_mode == "grid":
             if (
-                "enabled_modules" in session
-                and len(session.get("enabled_modules", [])) == 0
+                "enabled_module_indices" in session
+                and len(session.get("enabled_module_indices", []).get("JobContext"))
```

Comment:
If the first `get` returns the default value of `[]`, this will call `[].get("JobContext")`, which raises an `AttributeError`. Try:
```suggestion
                and len(session.get("enabled_module_indices", {}).get("JobContext", []))
```

Reply:
Also you probably don't have to check `"enabled_module_indices" in session` first, because you're doing that implicitly with the `.get(key, default)` syntax.

Reply:
This does result in real-world errors. I just triggered one for the "plots" example. I had no project modules, and I went from the "Jobs" page to the "Project" page and it failed.

```
  File "/home/bdice/code/signac-dashboard/signac_dashboard/dashboard.py", line 408, in _render_project_view
    and len(session.get("enabled_module_indices", []).get("ProjectContext"))
TypeError: object of type 'NoneType' has no len()
```